pulp_2to3_migration/app/migration.py

```diff
@@ -303,6 +344,7 @@ def create_repo_version(migrator, pulp2_repo, pulp3_remote=None):
         pulp2_repo.pulp3_repository_remote = pulp2_repo.pulp2importer.pulp3_remote
     if pulp2_repo.is_migrated:
         pulp2_repo.save()
+        progress_rv.update(total=F('total') - 1)
```

Comment:
Do you think the code would be improved by a helper on ```GroupProgressReport``` that does something like this?  (It's not necessary or even a suggestion, just an idea)

```python
def increment_by(self, field, amount):
    attrs = {field: F(field) + amount}
    self.update(attrs)
```
The API would be more similar to ```TaskGroup```

Reply:
@dralley at first i thought it is a good idea, but the thing is the update() is performed on the queryset, so i have just added the TaskGroup.current() helper